### PR TITLE
XS✔ ◾ update allure doc command SA-1847

### PR DESCRIPTION
**Pull request by mustafaStakater**
update allure doc command

fixes : https://stakater-cloud.atlassian.net/browse/SA-1847?atlOrigin=eyJpIjoiNzE4OTE1YThiNzBhNDFmYmI4OGU3NmEwOGFhOGU3NDkiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by stakater-user** [comment]
@mustafaStakater Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-135-b46b9d57`

**Comment by stakater-user** [comment]
@mustafaStakater Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-135-92a81c05`

**Comment by stakater-user** [comment]
@mustafaStakater Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-135-5ee77923`

**Comment by stakater-user** [comment]
@mustafaStakater Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-135-168b7035`

File: docs/3-revenge-of-the-automated-testing/2b-tekton.md [hunk 1]
```diff
@@ -158,8 +159,13 @@ Open up the console and navigate to your pipeline definition by going to `Pipeli
...
 
-Please append `/allure-docker-service/projects/stakater-nordmart-review/reports/latest/index.html` to your allure route urls to see the results once the task succeeds
+    Please append `/allure-docker-service/projects/stakater-nordmart-review/reports/latest/index.html` to your allure route urls to see the results once the task succeeds
 
-![allure-ui-Nordmart](./images/allure-ui-nordmart.png)
+    ![allure-ui-Nordmart](./images/allure-ui-nordmart.png)
 
+    Alternatively, You can get your Allure URL by running the following commands :
+    ```
+    TENANT_NAME=<TENANT_NAME>
```
Comment (AsfaMumtaz) [inline docs/3-revenge-of-the-automated-testing/2b-tekton.md:168]: Move this up and add the other thing as an alternative.

File: docs/3-revenge-of-the-automated-testing/2b-tekton.md [hunk 2]
```diff
@@ -158,8 +159,13 @@ Open up the console and navigate to your pipeline definition by going to `Pipeli
...
+    echo https://$(oc get route ${TENANT_NAME}-dev-allure --template='{{ .spec.host }}' -n ${TENANT_NAME}-dev)/allure-docker-service/projects/default/reports/latest/index.html
```
Comment (AsfaMumtaz) [inline docs/3-revenge-of-the-automated-testing/2b-tekton.md:169]: The url should have stakater-nordmart-review instead of 'default'

**Comment by stakater-user** [comment]
@mustafaStakater Image is available for testing. `docker pull stakater/workshop-exercise:SNAPSHOT-PR-135-46349828`

**Pull request merged**